zappa/core.py

```diff
@@ -1483,34 +1487,18 @@ def rollback_lambda_function_version(
 
         return response["FunctionArn"]
 
-    def is_lambda_function_ready(self, function_name):
-        """
-        Checks if a lambda function is active and no updates are in progress.
-        """
-        response = self.lambda_client.get_function(FunctionName=function_name)
-        return (
-            response["Configuration"]["State"] == "Active"
-            and response["Configuration"]["LastUpdateStatus"] != "InProgress"
```

Comment:
Do we know if the `waiter.wait()` will exactly mimic ^ this behaviour? My memory is a bit fuzzy, but when I first made this I think I just had the `response["Configuration"]["State"] == "Active"` check, and then had to later add the `and response["Configuration"]["LastUpdateStatus"] != "InProgress"` for some reason. I'm wondering if `waiter.wait()` would only check for the first condition and not necessarily the second

Reply:
Nice catch, I think you are correct in that the waiter only checks the State.

I've just checked [the source](https://github.com/boto/botocore/blob/23ee17f5446c78167ff442302471f9928c3b4b7c/botocore/data/lambda/2015-03-31/waiters-2.json#L21-L46) and it looks like we'd need a `function_updated` waiter too.